routing/dht/dht.go

```diff
@@ -76,11 +76,6 @@ func NewDHT(ctx context.Context, h host.Host, dstore ds.ThreadSafeDatastore) *Ip
 		return nil
 	})
 
-	// sanity check. this should **never** happen
-	if len(dht.peerstore.Addresses(dht.self)) < 1 {
```

Comment:
this was removed. panicing if it's > 100 wouldnt do anything here as this is on initialization. we need to solve that problem but it's not here :(